test/k8s-integration/main.go

```diff
@@ -270,6 +272,14 @@ func handle() error {
 		testDir = k8sDir
 	}
 
+	if *deploymentStrat == "gke" {
+		gkeRegional := isRegionalGKECluster(*gceZone, *gceRegion)
+		if *isRegionalCluster && !gkeRegional {
```

Comment:
What if we enforce that only one of `gceZone` and `gceRegion` could be set? Then we know the cluster is regional if `len(*gceRegion) > 0` and we don't need the `--is-regional-cluster` flag

Reply:
We still need --is-regional-cluster for when you're using a pre-existing cluster---you don't set the region or zone in that case and just use your current kubeconfig.

Reply:
Note that we already check that only one of these is set in main.go:112

Reply:
Gotcha sounds good

nit: maybe make this field targeted to pre-existing clusters only, i.e. only make the flag settable if `gceZone` and `gceRegion` are unset, and update the flag description. Otherwise it could get confusing for someone new to understand when `is-regional-cluster` should be set just from looking at the flag description

Reply:
Good idea will do.

Reply:
(added in the bringupCluster check)